products/BellHybrid/apps/application-bell-main/windows/BellBatteryStatusWindow.cpp

```diff
@@ -113,10 +112,20 @@ namespace gui
         if (data != nullptr) {
             const auto &switchData = static_cast<Data &>(*data);
             const auto image       = battery_utils::getBatteryLevelImage(batteryEntries, switchData.chargeLevel);
-            if (image) {
-                center->setImage(image->data(), imageType);
+            if (switchData.isItLowBatteryWaring) {
+                topDescription->setText(utils::translate("battery_low"));
+                std::string tempBotDesc = utils::translate("battery_remaining");
```

Comment:
I'd completely get rid of this variable and just put `utils::translate("battery_remaining")` directly in the function call